.hecat/url-check.yml

```diff
@@ -25,3 +25,10 @@ steps:
         - '^https://crates.io/crates/vigil-server$' # hecat/python-requests bug? always returns 404 but the website works in a browser
         - '^https://nitter.net$' # always times out from github actions but the website works in a browser
         - '^https://demo.memories.gallery/apps/memories/$' # always returns 401
+        - '^https://tuleap.net/plugins/git/tuleap/tuleap/stable?p=tuleap%2Fstable.git&a=tree' # site takes a while to load,  times out with the default 10s timeout
+        - '^https://www.admidio.org/' # always times out from github actions but the website works in a browser
+        - '^https://demo.aimeos.org/' # always times out from github actions but the website works in a browser
+        - '^https://appflowy.io/' # Cannot get certificate but loads fine in browser and shows as secure
+        - '^https://metronome.im/' # Failed to establish connection but loads fine in browser
```

Comment:
It used to return 200 not so long ago, so I don't think it should be added. This was probably a temporary error.